SETUP.md

```diff
@@ -231,15 +231,21 @@ docker exec -it local.spark-master /bin/bash
 
 Start `spark-shell` with the following command: Available users are `openhouse` and `u_tableowner`.
 ```
-bin/spark-shell --packages org.apache.iceberg:iceberg-spark-runtime-3.1_2.12:1.2.0   \
+export AWS_REGION=us-east-1
+bin/spark-shell --packages org.apache.iceberg:iceberg-spark-runtime-3.1_2.12:1.2.0,software.amazon.awssdk:bundle:2.20.18,software.amazon.awssdk:url-connection-client:2.20.18   \
   --jars openhouse-spark-runtime_2.12-*-all.jar  \
   --conf spark.sql.extensions=org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions,com.linkedin.openhouse.spark.extensions.OpenhouseSparkSessionExtensions   \
   --conf spark.sql.catalog.openhouse=org.apache.iceberg.spark.SparkCatalog   \
   --conf spark.sql.catalog.openhouse.catalog-impl=com.linkedin.openhouse.spark.OpenHouseCatalog     \
   --conf spark.sql.catalog.openhouse.metrics-reporter-impl=com.linkedin.openhouse.javaclient.OpenHouseMetricsReporter    \
   --conf spark.sql.catalog.openhouse.uri=http://openhouse-tables:8080   \
   --conf spark.sql.catalog.openhouse.auth-token=$(cat /var/config/$(whoami).token) \
-  --conf spark.sql.catalog.openhouse.cluster=LocalHadoopCluster
+  --conf spark.sql.catalog.openhouse.cluster=LocalFSCluster  \
+  --conf spark.sql.catalog.openhouse.com.linkedin.openhouse.relocated.io-impl=org.apache.iceberg.aws.s3.S3FileIO  \
```

Comment:
this is a bug in relocations in java-runtime should be `spark.sql.catalog.openhouse.io-impl` instead.